spec/services/monthly_statistics_timetable_spec.rb

```diff
@@ -33,7 +37,7 @@
 
     context 'when today is after the publishing date in the current month' do
```

Comment:
maybe update the context description to be 'when today is on or after the publishing date...'?

Reply:
Done!